File: lib/src/main/java/com/darklycoder/wifitool/lib/type/WiFiConnectType.java
package com.darklycoder.wifitool.lib.type;

/**
 * WiFi连接状态
 */
public enum WiFiConnectType {

    DISCONNECTED(0, "未连接"),
    CONNECTING(1, "连接中"),
    CONNECTED(2, "已连接");

    public int type;
    public String state;

    WiFiConnectType(int type, String state) {
        this.type = type;
        this.state = state;
    }

}

File: lib/src/main/java/com/darklycoder/wifitool/lib/type/WiFiConnectType.kt
package com.darklycoder.wifitool.lib.type

/**
 * WiFi连接状态
 */
enum class WiFiConnectType private constructor(var type: Int, var state: String) {

    DISCONNECTED(0, "未连接"),
    CONNECTING(1, "连接中"),
    CONNECTED(2, "已连接")

}
